### ISSUE TITLE
Rocker A & B are swapped

### ISSUE THREAD

**Issue by davemclaughlin** [industrialinternet/node-red-nodes-enocean-serial#1]
I have tested with 3 different switches and msg.rocker returns the wrong rocker. They are swapped.

Should be:

if(swState == "10" || swState == "30"){msg.rocker= "A";}
if(swState == "50" || swState == "70"){msg.rocker= "B";}

**Comment by industrialinternet** [comment]
@davemclaughlin Dave sorry haven't done anything with this for 2 yrs
Now using https://www.npmjs.com/package/node-enocean
With a light weight wrapper for Node-RED.
I can share the Ne wrapper but it needs a bit of tidy up.

**Comment by davemclaughlin** [comment]
Yes, I'd be interested to see how this works. I have it installed but not yet tried to use it. 

I am seeing some issues with the current one where I don't see the down state, only the up so keen to know if this is software or maybe the fact it does not get detected by the Enocean Pi itself.

dave@axoninstruments.biz

**Comment by davemclaughlin** [comment]
Any chance to get a look at the wrapper, complete or not as I am seeing some issues with this one still missing the down state messages? Need a happier wife :)

**Comment by industrialinternet** [comment]
Dave created a new repository 
https://github.com/industrialinternet/nr_wraper_node
